shared/src/test/scala/spray/json/CompactPrinterSpec.scala

```diff
@@ -39,9 +39,6 @@ class CompactPrinterSpec extends Specification {
     "print JsNumber(1.23) to '1.23'" in {
       CompactPrinter(JsNumber(1.23)) mustEqual "1.23"
     }
-    "print JsNumber(-1E10) to '-1E10'" in {
```

Comment:
ScalaJS' implementation of StringBuilder doesn't use scientific notation. We can split this test case for ScalaJS and JVM if we really need to; or just drop it.